lib/httpMethods.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
```

Comment:
Maybe it's would be better to split the supported methods into 2 arrays. it helps when to like to provide an option endable/disable for [WebDAV](https://www.ibm.com/docs/en/i/7.1?topic=concepts-webdav) methods.

```js

const baseHtttpMethods = ['DELETE', 'GET', 'HEAD', 'PATCH', 'POST', 'PUT', 'OPTIONS']
const webDAVHtttpMethods = ['PROPFIND', 'PROPPATCH', 'MKCOL', 'COPY', 'MOVE', 'LOCK', 'UNLOCK', 'TRACE', 'SEARCH']

module.exports = {
  supportedMethods: [...baseHtttpMethods, ...webDAVHtttpMethods] // you can use concat or whatever!
}

```